client/src/webview/localize.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// `localize` provides functionality to use localized strings in react webviews
+// This expects for translations to be defined in a script block with an id of
+// `l10-messages`.
+let localizedTerms = null;
+const localize = (term: string) => {
```

Comment:
what about parameter substitution

Reply:
Included in https://github.com/sassoftware/vscode-sas-extension/pull/1622/commits/8ed259f8e6bcdd1304d86cd465b81a63d7d15428